commit 7b9f63f71730954ed3c28fdee559e50e998d40e3 (refs/changes/50/350/1)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2016-08-25T23:05:12+00:00 (3 years, 1 month ago)
    
    OS-5626 illumos-kvm needs to do its check_rtime part

diff --git a/check_rtime.ex b/check_rtime.ex
new file mode 100644
index 0000000..fb78f63
--- /dev/null
+++ b/check_rtime.ex
@@ -0,0 +1,23 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Joyent, Inc.
+#
+
+#
+# Exceptions for check_rtime logic. This list should be whittled down
+# over time. For an explanation, see check_rtime(1ONBLD) or
+# exception_lists/check_rtime in illumos-joyent.
+#
+
+SKIP	^usr/lib/devfsadm/linkmod/JOY_kvm_link.so$
+SKIP	^usr/lib/mdb/MACH(kvm)/kvm.so$
